e2e/components/Accordion/Accordion-test.avt.e2e.js

```diff
@@ -48,6 +48,19 @@ test.describe('@avt Accordion', () => {
     );
   });
 
+  test('@avt-advanced-states controlled', async ({ page }) => {
+    await visitStory(page, {
+      component: 'Accordion',
+      id: 'components-accordion--controlled',
+      globals: {
+        theme: 'white',
+      },
+    });
+    await expect(page).toHaveNoACViolations(
+      'Accordion @avt-advanced-states controlled'
+    );
+  });
+
```

Comment:
This story is already so close to the original story that already has toHaveNoACViolations, I think this can be removed.
```suggestion
```